src/plugin-sidebar.js

```diff
@@ -58,6 +58,23 @@ import AboutPlugin from './editor-sidebar/about';
 import ResetTheme from './editor-sidebar/reset-theme';
 import './plugin-styles.scss';
 
+function PluginSidebarItem( { icon, path, children, ...props } ) {
```

Comment:
I created a common component whose behavior changes depending on whether its items are navigable or not.